content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md

```diff
@@ -63,18 +63,18 @@ Based on your user license count, we recommend these machine types.
 GCE virtual machines are created as a member of a network, which has a firewall. For the network associated with the {% data variables.product.prodname_ghe_server %} VM, you'll need to configure the firewall to allow the required ports listed in the table below. For more information about firewall rules on Google Cloud Platform, see the Google guide "[Firewall Rules Overview](https://cloud.google.com/vpc/docs/firewalls)."
 
 1. Using the gcloud compute command-line tool, create the network. For more information, see "[gcloud compute networks create](https://cloud.google.com/sdk/gcloud/reference/compute/networks/create)" in the Google documentation.
-  ```shell
-  $ gcloud compute networks create <em>NETWORK-NAME</em> --subnet-mode auto
-  ```
+    ```shell
+    $ gcloud compute networks create <em>NETWORK-NAME</em> --subnet-mode auto
+    ```
 2. Create a firewall rule for each of the ports in the table below. For more information, see "[gcloud compute firewall-rules](https://cloud.google.com/sdk/gcloud/reference/compute/firewall-rules/)" in the Google documentation.
-  ```shell
-  $ gcloud compute firewall-rules create <em>RULE-NAME</em> \
-  --network <em>NETWORK-NAME</em> \
-  --allow tcp:22,tcp:25,tcp:80,tcp:122,udp:161,tcp:443,udp:1194,tcp:8080,tcp:8443,tcp:9418,icmp
-  ```
-  This table identifies the required ports and what each port is used for.
-
-  {% data reusables.enterprise_installation.necessary_ports %}
+    ```shell
+    $ gcloud compute firewall-rules create <em>RULE-NAME</em> \
+    --network <em>NETWORK-NAME</em> \
+    --allow tcp:22,tcp:25,tcp:80,tcp:122,udp:161,tcp:443,udp:1194,tcp:8080,tcp:8443,tcp:9418,icmp
+    ```
+    This table identifies the required ports and what each port is used for.
```

Comment:
Looks like this table isn't rendering as expected in stage:
https://docs-675--code-fence-indents.herokuapp.com/en/enterprise-server@2.22/admin/installation/installing-github-enterprise-server-on-google-cloud-platform#configuring-the-firewall

Here's what it looks like for me:
![image](https://user-images.githubusercontent.com/54248166/98895905-890e4a00-24f3-11eb-99b0-cf6fe2df2803.png)

For comparison with prod:
https://docs.github.com/en/enterprise-server@2.22/admin/installation/installing-github-enterprise-server-on-google-cloud-platform#configuring-the-firewall

Reply:
OK, went back to 3 spaces and it seems happy again.
I'm finding the markdown process a little frustrating, as it doesn't seem to follow CommonMark or GFM conventions

Reply:
Thanks for the fix! 🎉